routes.go

```diff
@@ -282,11 +280,9 @@ func provideURLPrefix(in provideURLPrefixIn) string {
 }
 
 //nolint:funlen
-func fixV2Duration(getLogger ancla.GetLoggerFunc, config ancla.TTLVConfig, v2Handler http.Handler) (alice.Constructor, error) {
+func fixV2Duration(getLogger sallust.GetLoggerFunc, config ancla.TTLVConfig, v2Handler http.Handler) (alice.Constructor, error) {
```

Comment:
The `sallust.GetLoggerFunc` type has been deprecated and removed.  You can just declare the function type inline, or reorganize things so that this isn't passed as a parameter.

Reply:
yup this is already patched in https://github.com/xmidt-org/tr1d1um/pull/344